source/linux/How_to_Guides/Target/How_to_fix_y2k38.rst

```diff
@@ -0,0 +1,43 @@
+.. include:: /_replacevars.rst
+
+How to Fix Y2K38 Issue
+======================
+
+What is the Y2K38 Issue?
+------------------------
+
+The Y2K38 issue is a bug which disallows a date from exceeding 03:14:07 UTC on January 19, 2038, on 32-bit systems. This
+is because the structures used to store the date experience an integer overflow when the date is set beyond the mentioned
+one. This causes the system to crash.
+For more information, see: https://en.wikipedia.org/wiki/Year_2038_problem
```

Comment:
If you want this to be rendered on a newline you need an additional carriage return

Reply:
Addressed your comments @StaticRocket